tests/span_tests.cpp

```diff
@@ -158,27 +153,27 @@ TEST(span_test, from_pointer_length_constructor)
     int arr[4] = {1, 2, 3, 4};
 
     {
-        for (int i = 0; i < 4; ++i)
+        for (std::size_t i = 0; i < 4; ++i)
```

Comment:
Technically you've included `<stddef.h>` for `size_t` but not `<cstddef>` for `std::size_t`.

Reply:
I'll switch this to <cstddef> instead of <stddef.h>, it was indirectly being pulled in through <gsl/span.